tests/unit/gaze/_utils/_parsing_test.py

```diff
@@ -45,7 +45,7 @@
 lines
 MSG	2154555 RECCFG CR 1000 2 1 L
 MSG	2154555 ELCLCFG BTABLER
-MSG	2095865 DISPLAY_COORDS 0 0 1279 1023
```

Comment:
instead of replacing this, just add a second line with the `GAZE_COORDS`

Reply:
Added it before the RECCFG (where it is usually logged)